src/main/java/org/cancogenvirusseq/muse/service/SubmissionService.java

```diff
@@ -65,6 +61,13 @@ public SubmissionService(@NonNull SubmissionRepository submissionRepository) {
 
   public Mono<SubmissionCreateResponse> submit(
       Flux<FilePart> fileParts, SecurityContext securityContext) {
+    val userId =
+        securityContext != null
+                && securityContext.getAuthentication() != null
+                && securityContext.getAuthentication().getName() == null
+            ? UUID.fromString(securityContext.getAuthentication().getName())
+            : UUID.randomUUID();
```

Comment:
will always be secure, need to update security config to not allow requests without JWT